README.md

```diff
@@ -95,9 +79,14 @@ export class AppComponent {
 }
 ```
 
-## Configuration Options
+## API reference
```

Comment:
Maybe in lieu of an actual docs site we could move these to an `API.md` document and reference that to keep the readme lighter?